cl/corpus_importer/management/commands/tn_work_comp.py

```diff
@@ -0,0 +1,159 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+from datetime import datetime
+
+import requests
+from django.core.exceptions import ValidationError
+from django.utils.encoding import force_bytes
+
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.crypto import sha1
+from cl.people_db.models import Person
+from cl.scrapers.management.commands.cl_scrape_opinions import (
+    make_objects,
+    save_everything,
+)
+from cl.scrapers.tasks import extract_doc_content
+from cl.search.models import Court, Opinion, Docket
+
+
+def process_date(date_str):
```

Comment:
See if you can find code that does this elsewhere in the codebase? Look for dateutil?